packages/paper/src/paper.ts

```diff
@@ -74,6 +74,10 @@ export class Paper<UserOperations extends OperationMap = OperationMap> {
     return findDocument(this.data, documentOrKey);
   }
 
+  truncate(): void {
+    this.current = createDocumentStore(this.sourceGraphQLSchema);
```

Comment:
I considered a few names:
* `clear`
* `clean`
* `empty`
* `reset`
* `truncate`

I think `clear` might be the best, most concise discoverable name for this. Can you also create a new array for the `history`, as the `current` state is only related as a series of changes captured in `history`.

Reply:
I think in the database context, `truncate` is more intuitive and familiar terminology. That's why I choose that. But if you feel `clear` works better, we can rename it.

I've updated the method to clear the history as well. Thanks.

Reply:
I'm not set on `clear` and I do agree `truncate` makes perfect sense with dbs. I guess I'm trying to treat paper more as an in-memory store, a structured object in memory and less to do with "disk" where I think truncate also makes sense.

Reply:
Your call.

Reply:
Alright, lets go with `clear`.

Reply:
done.